packages/react/src/Popover/Popover.tsx

```diff
@@ -54,11 +54,32 @@ const Popover = React.forwardRef<HTMLDivElement, PopoverProps>(function Popover(
 })
 Popover.displayName = 'Popover'
 
-export type PopoverContentProps = {className?: string} & StyledPopoverProps & HTMLProps<HTMLDivElement>
+export type PopoverContentProps = {
+  className?: string
+  width?: 'small' | 'large' | 'medium' | 'auto' | 'xlarge' | 'xxlarge'
+  height?: 'xsmall' | 'small' | 'large' | 'medium' | 'auto' | 'xlarge' | 'fit-content'
+  overflow?: 'auto' | 'hidden' | 'scroll' | 'visible'
```

Comment:
Let's add JSDoc comments and update `packages/react/src/Popover/Popover.docs.json` to add these props.

Reply:
Updating packages/react/src/Popover/Popover.docs.json should do it!